spec/phonegap/create.spec.js

```diff
@@ -70,183 +257,25 @@ describe('phonegap.create(options, [callback])', function() {
+        ** callback test moved to ../prespec/create.spec.js
+        'when my-app/www/config.xml exists should move it to my-app/config.xml'
+        'when my-app/www/config.xml does not exist should not move it to my-app/config.xml'
+        'when updating config.xml should parse the my-app/config.xml'
+        'when config.xml does not exist should trigger a "warn" event'
+        'when --link-to is provided should not move config.xml should not update config.xml'
+        'when complete should trigger callback without an error'
+         */
+    //});
 
-        describe('when my-app/www/config.xml does not exist', function() {
-            beforeEach(function() {
-                fs.existsSync.andReturn(false);
-            });
-
-            it('should not move it to my-app/config.xml', function(done) {
-                phonegap.create(options, function(e) {
-                    expect(fs.existsSync).toHaveBeenCalled();
-                    expect(fs.renameSync).not.toHaveBeenCalled();
-                    done();
-                });
-            });
-        });
-
-        describe('when config.xml does not exist', function() {
-            beforeEach(function() {
-                fs.existsSync.andReturn(false);
-            });
-
-            it('should trigger a "warn" event', function(done) {
-                phonegap.on('warn', function(message) {
-                    expect(message).toMatch('could not update');
-                    done();
-                });
-                phonegap.create(options, function(e) {});
-            });
-        });
-
-        describe('when --link-to is provided', function() {
-            beforeEach(function() {
-                options['link-to'] = '/path/to/app';
-            });
-
-            it('should not move config.xml', function(done) {
-                phonegap.create(options, function(e) {
-                    expect(fs.existsSync).not.toHaveBeenCalled();
-                    expect(fs.renameSync).not.toHaveBeenCalled();
-                    done();
-                });
-            });
-        });
-
-        describe('when complete', function() {
-            it('should trigger callback without an error', function(done) {
-                phonegap.create(options, function(e) {
-                    expect(e).toBeNull();
-                    done();
-                });
-            });
-        });
-    });
-
-    describe('failed to create a project', function() {
-        beforeEach(function() {
-            phonegap.cordova.andCallFake(function(options, callback) {
-                callback(new Error('path already exists'));
-            });
-        });
 
-        it('should trigger callback with an error', function(done) {
-            phonegap.create(options, function(e) {
-                expect(e).toEqual(jasmine.any(Error));
-                done();
-            });
+    it('should trigger callback with an error because cordova-create is not mocked', function(done) {
```

Comment:
I would remove this test for the purposes of this PR. We can always add future test coverage to, at a finer level, exercise the various failure modes.